gstreamer-rtsp-server/src/auto/r_t_s_p_session.rs

```diff
@@ -0,0 +1,229 @@
+// This file was generated by gir (37f7805+) from gir-files (???)
+// DO NOT EDIT
+
+use RTSPMedia;
+use RTSPSessionMedia;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPSession(Object<ffi::GstRTSPSession, ffi::GstRTSPSessionClass>);
+
+    match fn {
+        get_type => || ffi::gst_rtsp_session_get_type(),
+    }
+}
+
+impl RTSPSession {
+    pub fn new(sessionid: &str) -> RTSPSession {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib_full(ffi::gst_rtsp_session_new(sessionid.to_glib_none().0))
+        }
+    }
+}
+
+unsafe impl Send for RTSPSession {}
+unsafe impl Sync for RTSPSession {}
+
+pub trait RTSPSessionExt {
+    fn allow_expire(&self);
+
+    //fn filter<'a, P: Into<Option<&'a /*Unimplemented*/RTSPSessionFilterFunc>>, Q: Into<Option</*Unimplemented*/Fundamental: Pointer>>>(&self, func: P, user_data: Q) -> Vec<RTSPSessionMedia>;
```

Comment:
All the filter functions are quite useful for applications (also from the other objects). Something for another PR though